src/Core/src/Handlers/Image/ISetImageHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿#if __IOS__ || MACCATALYST
+using PlatformImage = UIKit.UIImage;
+#elif MONOANDROID
+using PlatformImage = Android.Graphics.Drawables.Drawable;
+#elif WINDOWS
+using PlatformImage = Microsoft.UI.Xaml.Media.ImageSource;
+#elif TIZEN
+using PlatformImage = Microsoft.Maui.Platform.MauiImageSource;
+#elif (NETSTANDARD || !PLATFORM) || (NET6_0_OR_GREATER && !IOS && !ANDROID && !TIZEN)
+using PlatformImage = System.Object;
+#endif
+
+namespace Microsoft.Maui
+{
+	public interface ISetImageHandler : IElementHandler
```

Comment:
Maybe this can follow the "IImageSourcePart*" and be a `IImageSourcePartSetter` or somthing to fit in as this really is just a component of the bigger `ImageSourcePartLoader` "system".